core/src/main/scalajvm/scalapb/zio_grpc/server/ZServerCallHandler.scala

```diff
@@ -39,6 +39,11 @@ class ZServerCallHandler[R, Req, Res](
 }
 
 object ZServerCallHandler {
+  private[zio_grpc] val queueSizeProp = "zio-grpc.backpressure-queue-size"
+
+  val backpressureQueueSize: UIO[Int] =
```

Comment:
It looks like this would silently default to 16 if the input is invalid, and this would make things very confusing to our users since they'd believe they run it under a certain setting. `toInt` is fairly particular - it would fail even if there's whitespace after the number. I think a better experience would be to fail the whole program if there's an exception here, it would save users a lot of time later.